available_images.md

```diff
@@ -409,21 +302,4 @@ Prior Habana Training Container Versions
 |PyTorch 1.10.1     |training   |HPU            | 3.8 (py38)            |1.3.0              |763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training-habana:1.10.1-hpu-py38-synapseai1.3.0-ubuntu20.04 |
 |TensorFlow 2.8.0   |training   |HPU            | 3.8 (py38)            |1.3.0              |763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-training-habana:2.8.0-hpu-py38-synapseai1.3.0-ubuntu20.04 |
 |PyTorch 1.10.0     |training   |HPU            | 3.8 (py38)            |1.2.0              |763104351884.dkr.ecr.us-east-1.amazonaws.com/pytorch-training-habana:1.10.0-hpu-py38-synapseai1.2.0-ubuntu20.04            |
```

Comment:
I think these 3 are out of support as well. We just support last 2 DLC versions for Habana

Reply:
removed them as well, thanks